src/mame/philips/patgen.cpp

```diff
@@ -0,0 +1,418 @@
+	ROM_LOAD( "4008_002_00621_csum_06e1.v301",  0x000000, 0x80000, CRC(25a095d6) SHA1(ba7d37683b8fa77122fa4b2b1df31911634254d7) )
+	ROM_LOAD( "4008_002_00631_csum_ffb8.v303",  0x080000, 0x80000, CRC(af031008) SHA1(6dbaba28c9f959d8594fc0b8b874ba6e0a83edbf) )
+
+	ROM_REGION( 0x17000, "pal", 0 )
+	ROM_LOAD( "pal_4008_002_00641_csum_e123.v37.jed",  0x000000, 0x01c45, CRC(172ec12f) SHA1(2ab12bc8024c2a0b4ea7a0f8aa706e3293e366eb) )
+	ROM_LOAD( "pal_4008_002_00651_csum_f80d.v38.jed",  0x002000, 0x01c45, CRC(3aef572a) SHA1(0aa4d946b91550b102f7f098fe1484d2b226a017) )
+	ROM_LOAD( "pal_4008_102_56171_csum_18d9.v205.jed",  0x004000, 0x01c46, CRC(5a2d22a2) SHA1(f274e38aac896b802eb44f1004d72b9d8d090eef) )
+	ROM_LOAD( "pal_4008_102_56181_csum_10a1.v305.jed",  0x006000, 0x01c46, CRC(2e331f24) SHA1(41ceb44b6af43abc4a19c679255fbc763e68e2c5) )
+	ROM_LOAD( "pal_4008_102_58291_csum_6653.v310.jed",  0x008000, 0x01c46, CRC(60704fbf) SHA1(c1fa652927a610438f4eabffcbcf80922fb98e6b) )
+	ROM_LOAD( "pal_4008_102_58791_csum_681f.v40.jed",  0x010000, 0x01c45, CRC(f78297e1) SHA1(bb6c5430f66eb02f3133e95d7db268d4977a45c1) )
+	ROM_LOAD( "pal_4008_102_59681_csum_2c6d.v39.jed",  0x012000, 0x01c45, CRC(f0921c84) SHA1(81cb2f8aa5651c47b79ec25a852d6fb78e9811fe) )
+	ROM_LOAD( "peel_4008_102_55441_csum_4a64.v15.jed",  0x014000, 0x00dc5, CRC(cbd86564) SHA1(21b8268ba93a52f9efe0f481276a2862d4fb58dd) )
+	ROM_LOAD( "peel_4008_102_55661_csum_6055.v111.jed",  0x016000, 0x00dc5, CRC(3418434f) SHA1(2570d8f2138ae9ca182ac4ac6f8fe9b0973f4bb9) )
+ROM_END
+
+ROM_START( pm5644l00 )
+	ROM_REGION( 0x10000, "maincpu", 0 )
+	ROM_LOAD( "4008_002_01671_csum_cd00.v12",  0x000000, 0x10000, CRC(f06acb17) SHA1(9e7da23c3d6624190561cfc89f86fb5c9b563a63) )
+
+	ROM_REGION( 0x280000, "luma", 0 )
+	ROM_LOAD( "4008_002_01621_csum_c923.v101",  0x000000, 0x80000, CRC(5b2834d9) SHA1(483d285445e935cdbd05e4fef0e1c2cb5ce890c8) )
+	ROM_LOAD( "4008_002_01631_csum_00d6.v103",  0x080000, 0x80000, CRC(ba4fb4d7) SHA1(acf357b0ffea7bdd3259dae3fc9292dcb799e481) )
+	ROM_LOAD( "4008_002_01641_csum_c9d6.v105",  0x100000, 0x80000, CRC(b4a6e5c7) SHA1(329a93a8466eee4260e97a15c514d5da01211019) )
+	ROM_LOAD( "4008_002_01651_csum_9c03.v107",  0x180000, 0x80000, CRC(2cd4931f) SHA1(1270f8ea23a68c14b2373a05dab394921c4ca11d) )
+	ROM_LOAD( "4008_002_01661_csum_00e2.v109",  0x200000, 0x80000, CRC(1a85ff7b) SHA1(edab58f288f8be70a25488a38fe80ac899d9c21b) )
+
+	ROM_REGION( 0x15000, "pal", 0 )
+	ROM_LOAD( "pal_4008_102_56881_csum_0483.v37.jed",  0x000000, 0x01c46, CRC(c9e21167) SHA1(6a21419a0aa8f22b45cf03845c2e42d4ab4c28bf) )
+	ROM_LOAD( "pal_4008_102_56891_csum_8669.v38.jed",  0x001c46, 0x01c46, CRC(4b6be56d) SHA1(927fa0a6d3de55ec141df1ac5b76d1331f5175fe) )
+	ROM_LOAD( "pal_4008_102_56901_csum_72e4.v40.jed",  0x00388c, 0x01c45, CRC(354a13be) SHA1(d0b2965e3b24fa47fef345c9068c26118bf3bc45) )
+	ROM_LOAD( "peel_4008_102_56301_csum_4ed3.jed",  0x0054d1, 0x00dc5, CRC(2a81d3e0) SHA1(62aa1647522940233de1f2d58103ee74f36b7d97) )
+ROM_END
```

Comment:
Description for this set claims `SECAM 4:3 Colour Circle`, but I don't see UV roms here, what's the catch?